src/hooks/useDisputableVotes.js

```diff
@@ -61,18 +64,33 @@ export function useDisputableVote(proposalId) {
           `${disputableVotingApp.address}-vote-${proposalId}`
         )
 
-        const [collateral, settings] = await Promise.all([
+        const [collateral, settings, orgToken] = await Promise.all([
           vote.collateralRequirement(),
           vote.setting(),
+          vote.token(),
         ])
 
         const token = await collateral.token()
 
+        let voterInfo
+        if (account) {
+          const balance = await token.balance(account)
+          voterInfo = {
+            accountBalanceNow: formatTokenAmount(balance, orgToken.decimals),
+            accountBalance: await vote.formattedVotingPower(account),
+            hasVoted: await vote.castVote(account),
+            canExecute: await vote.canExecute(account),
+            canVote: await vote.canVote(account),
+          }
+        }
```

Comment:
Can we combine some of these with `Promise.all`? Don't worry too much about this if it's not too slow. I'd like to update this code shortly anyway with the polling changes.